handbook/product/roles/product_manager_engineering_manager_responsibilities.md

```diff
@@ -112,7 +112,7 @@ td {
 ### Reporting
 | EM expectations | PM expectations |
 | --- | --- |
-| - Provide weekly status report to engineering leadership | - Look over weekly update by EOD [the day before it gets sent] and supplement any info |
+| - Provide weekly status report to engineering leadership | - Provide weekly progress highlight to share at company meeting <br> - Look over weekly update by EOD [the day before it gets sent] and supplement any info |
```

Comment:
```suggestion
| - Provide weekly status report to engineering leadership | - Provide weekly progress highlight for weekly status report email that is also shared at the company meeting <br> - Look over weekly update by EOD [the day before it gets sent] and supplement any info |
```

Reply:
@macraig worth specifying that this update goes in the email? Don't want new PMs/folks reading to this to wonder where the progress highlight is supposed to go.

Reply:
Since EM is writing the email, shouldn't this go under EM responsibilities?

Reply:
I think the "highlight" could fall under either PM or EM – both should be able to call out the most important thing their team worked on that week if the PM is communicating well – but since this is similar to "team goals" or "product marketing [even to internal stakeholders]" it felt more PM-related given the other responsibilities on this page. I may just be biasing towards how the Web team collaborates on these emails before they get sent, though, and don't think anything in this matrix should stop a team from doing something that they're finding effective.

Reply:
ok

Reply:
I agree with @Joelkw in that although the EM is accountable for writing and sending the update, being accountable for the one-sentence highlight suitable for all audiences feels more PM-related. That said, the whole process is a collaborative effort and each team should totally do whats works best for them.